src/app/shared/components/question/question.component.html

```diff
@@ -44,7 +44,7 @@ <h2 attr.data-selector="round-{{round.question_round_number}}" class="govuk-head
 
     <!-- Question in Sent state-->
     <ng-template #questionsSent>
-        <p data-selector="sent-questions-details">You sent {{round.questions.length}} {{round.questions.length <= 1 ? 'question' : 'questions'}} to the appellant {{round.questions[0].state_datetime | date: "'at' h:mmaaaaa'm 'on' d MMMM yyyy":'+00:00'}}</p>
+        <p data-selector="sent-questions-details">You sent {{round.questions.length}} {{round.questions.length <= 1 ? 'question' : 'questions'}} to the appellant {{round.questions[0].state_datetime | date: "'at' h:mmaaaaa'm 'on' d MMMM yyyy":'+00:00'}}.</p>
```

Comment:
If round.questions.length == 0 then  it will say
You sent  0 question
It should say you sent 0 questions
the comparator should be round.questions.length == 1